template/bin/source/wp-config-sample.php

```diff
@@ -18,13 +18,13 @@
  * @package WordPress
  */
 
-if ( ! file_exists( dirname( __DIR__ ) . '/vendor/autoload.php' ) ) {
+if ( ! file_exists( ABSPATH . '../../vendor/autoload.php' ) ) {
```

Comment:
Is it still working when you run the app from webserver?

Instead of updating `wp-config.php` file, I think you can use `--path` command argument to specify in which folder command need to run (see [arguments](https://make.wordpress.org/cli/handbook/references/config/#global-parameters)).

Reply:
I think the `path` option is already set by default via `wp-cli.yml` which contain `path: web/wp/`, i'm not sure how to test it's working

I recon that the solution provided in this PR, is kinda bad. But it's working on the webserver. I'm totally down to find a better way to handle this issue 😄

Reply:
I found the origin of the bug!

The WP CLI evaluates the code of the `wp-config.php` (see [Runner.php:1211](https://github.com/wp-cli/wp-cli/blob/master/php/WP_CLI/Runner.php#L1211)) file and search and replace each occurrence of the `__DIR__` and `__FILE__` constants with the correct path with the following regex: `/(?>\'[^\']*?\')|(?>"[^"]*?")|(?<file>\b__FILE__\b)|(?<dir>\b__DIR__\b)/m` (see [utils.php:36](https://github.com/wp-cli/wp-cli/blob/master/php/utils.php#L36) and [Runner.php:612](https://github.com/wp-cli/wp-cli/blob/master/php/WP_CLI/Runner.php#L612)).

I tested the content of our `wp-config.php` file and it appears that a single quote in the first comment of the files breaks the regex (see https://regex101.com/r/5BnTCY/1). Replacing this quote with a `’` fixes the bug (see https://regex101.com/r/5BnTCY/2), see the diff below:

```diff
diff --git a/bin/source/wp-config-sample.php b/bin/source/wp-config-sample.php
index fba8e0a..26daddf 100644
--- a/bin/source/wp-config-sample.php
+++ b/bin/source/wp-config-sample.php
@@ -3,7 +3,7 @@
  * The base configuration for WordPress
  *
  * The wp-config.php creation script uses this file during the
- * installation. You don't have to use the web site, you can
+ * installation. You don’t have to use the web site, you can
  * copy this file to "wp-config.php" and fill in the values.
  *
  * This file contains the following configurations:
```